### DISCUSSION TITLE
Request to change repository urls on two apps

### DISCUSSION THREAD

**Discussion by ghost** [codemagic-ci-cd/codemagic-docs#2711] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

We would like to update the repository URLs for two apps

First app:
New repository url: https://codemagic.io/app/66d72ee28609dd66f0873bb0/
Old repository url: https://codemagic.io/app/5eb9902aa6d15e6b6c37667d/

Second app:
New repository url: https://codemagic.io/app/66d72f0eac97639604c143d1/
Old repository url: https://codemagic.io/app/5eb9a0f1a6d15e63cbb05df9/

Our team id: https://codemagic.io/teams/66cdf4895f3c0ad4c86a608c

### Expected results

I would like to preserve the app from the old repository and just update the urls to the new location.

### Actual results

Can't change the repository urls in the ui.

**Comment by himesh-cm** [comment, marked as answer]
Hey @VinceERS, Please email us at info@codemagic.io from your registered email address with the above details. Someone from the team will contact you regarding your request.

**Comment by ghost** [comment]
Ok thank you I will do that.
